web/src/utils/Cryption.ts

```diff
@@ -0,0 +1,28 @@
+import CryptoJS from 'crypto-js'
+
+const keyValue:any = process.env.REACT_APP_ENCRYPTION_KEY
+const privateKeyValue:any = process.env.REACT_APP_ENCRYPTION_PRIVATE_KEY
+
+export function Encrypt(data:string) {
+    try {
+      const _key = CryptoJS.enc.Utf8.parse(keyValue);
+      const privatekey = CryptoJS.enc.Utf8.parse(privateKeyValue);
+      const encryptedData = CryptoJS.DES.encrypt(data, _key, { iv: privatekey }).toString();
+  
+      return encryptedData;
+    } catch (ex:any) {
+      return ex.message;
+    }
+}
+
+export function Decrypt(encryptedData:any) {
+  try {
```

Comment:
Why Try catch? you returning the exception mesasge as decrypted value? You know this can go wrong?